### PR TITLE
itR-11: feat: add internationalization to the project

### PR DESCRIPTION
**Pull request by MilaKolasava**

### CODE REVIEW

File: src/i18n/messages/es-US.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { LOCALES } from "../Locales";
+
+export default {
+  [LOCALES.ENGLISH]: {
+    hello: "ghgjkflsjah",
+    main: "main",
+    about: "about",
+    dark: "dark",
+    light: "light",
+
```
Comment (glebpigulevsky) [inline src/i18n/messages/es-US.js]: empty line

File: src/i18n/messages/be-BY.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { LOCALES } from "../Locales";
+
+export default {
+  [LOCALES.BELARUSIAN]: {
+    hello: "Прывiтанне",
+      main: "Галоўная",
```
Comment (glebpigulevsky) [inline src/i18n/messages/be-BY.js]: extra space

File: src/i18n/Translate.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import React from 'react'
```
Comment (glebpigulevsky) [inline src/i18n/Translate.js]: use useIntl, we dont need component

File: src/components/Header/Header.tsx [hunk 4]
```diff
@@ -33,6 +35,9 @@ function Header(props: HeadersProps) {
           setIsDarkThemeOff={props.setIsDarkThemeOff}
           isDarkThemeOff={props.isDarkThemeOff}
         />
+        <button onClick={() => props.setLocale(LOCALES.ENGLISH)}>EN</button>
+        <button onClick={() => props.setLocale(LOCALES.POLISH)}>PL</button>
+        <button onClick={() => props.setLocale(LOCALES.BELARUSIAN)}>BY</button>
```
Comment (glebpigulevsky) [inline src/components/Header/Header.tsx]: mode it please in itR-12 ticket

**Review by glebpigulevsky — CHANGES_REQUESTED** [review]
allowJs : false. tests!

File: src/App.tsx [hunk 5]
```diff
@@ -5,18 +5,25 @@ import Footer from "./components/Footer/Footer";
 import Header from "./components/Header/Header";
 import Jumbotron from "./components/Jumbotron/Jumbotron";
 import ModalWindow from "./components/ModalWindow/ModalWindow";
+import { I18nProvider, LOCALES } from "./i18n";
+import Translate from "./i18n/Translate";
 
 function App() {
   const [modalActive, setModalActive] = useState<boolean>(() => false);
   const [isDarkThemeOff, setIsDarkThemeOff] = useState<boolean>(() => true);
+  const [locale, setLocale] = useState(LOCALES.ENGLISH);
```
Comment (glebpigulevsky) [inline src/App.tsx]: itR-12

File: src/i18n/messages/es-US.js (hunk 1 above)
Comment (MilaKolasava) [reply src/i18n/messages/es-US.js]: done

File: src/i18n/messages/be-BY.js (hunk 2 above)
Comment (MilaKolasava) [reply src/i18n/messages/be-BY.js]: done

File: src/components/Header/Header.tsx (hunk 4 above)
Comment (MilaKolasava) [reply src/components/Header/Header.tsx]: done

File: src/i18n/Translate.js (hunk 3 above)
Comment (MilaKolasava) [reply src/i18n/Translate.js]: done

File: src/App.tsx (hunk 5 above)
Comment (MilaKolasava) [reply src/App.tsx]: I'm doing it

**Review by glebpigulevsky — APPROVED** [review]
вельмі добрая праца. малайчына

**Pull request merged**